scrapers/Xslist.yml

```diff
@@ -0,0 +1,65 @@
+name: "XsList (JAV)"
+performerByName:
+  action: scrapeXPath
+  queryURL: https://xslist.org/search?query={}&lg=en
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url: 
+      - xslist.org/
+    scraper: performerScraper
+
+xPathScrapers:
+  performerSearch:
+    performer:
+      Name: //li[@class="clearfix"]/h3/a/@title
+      URL: 
+        selector: //li[@class="clearfix"]/h3/a/@href
+  
+  performerScraper:
+    performer:
+      Name: //span[@itemprop="name"]/text()
+      Aliases: 
```

Comment:
Aliases seems weird `https://xslist.org/en/model/91.html` , `https://xslist.org/en/model/94586.html`
This looks better
```yaml
      Aliases:
        selector: //span[@itemprop="additionalName"]/text()
        concat: ", "
```

Reply:
Sometimes performer don't have this line, so your selector don't work. `https://xslist.org/en/model/5.html`
Manage to put your selector + mine with a new regex:

```      Aliases: 
        selector: //span[@itemprop="additionalName"]/text()|//div[@id="layout"]/div/h2[1]/text()
        concat: ", "
        postProcess:
          - replace: 
            - regex: \,\s\w+\s\w+\s\(.+
              with: 
            - regex: (.+\()(\D\W+)([\/)].+)
              with: $2```

Reply:
By weird I meant that it's not actually the alias but the name written in Kanji.
If you want to keep that too as an alias then imho your selector would need to come before mine in the union operator so that we end up with : name in kanji , alias1 , alias2 , alias3 ....

Reply:
Her name in Kanji can be useful so I like having it.
I made a new regex to get  `name in kanji , alias1 , alias2 , alias3 ...`.